frontend/src/metabase/query_builder/components/NativeQueryEditor/NativeQueryEditor.tsx

```diff
@@ -272,7 +390,7 @@ export class NativeQueryEditor extends Component {
       return;
     }
 
-    this._editor = ace.edit(editorElement);
+    this._editor = ace.edit(editorElement) as Ace.Editor;
```

Comment:
TypeScript is right to complain here - any method called in this function after this line could possibly change `this._editor` to `null`.

I think `checkNotNull` fits better than a type-cast here. But we'd need to assign `this._editor` to a variable.

```suggestion
    const editor = checkNotNull(ace.edit(editorElement));
```